.env_example

```diff
@@ -90,5 +90,8 @@ SCORE_RESPONSE_LIMIT=50
 # IDs of users whom appear to have everyone added as a friend (comma-separated)
 SUPER_FRIENDLY_USERS=1
 
+# Cutoff timestamp for showing "since the beginning" join dates
+BEGINNING_ENDED_AT=2023-12-31T06:00:00
```

Comment:
for reference the one osu used was 2007-12-31T06:00:00 , i just moved it forward to titanic's first end of year